packages/cli/src/globals.ts

```diff
@@ -0,0 +1,43 @@
+import { message, object, option } from "@optique/core";
+import { configure, getConsoleSink } from "@logtape/logtape";
+import { getFileSink } from "@logtape/file";
+import { recordingSink } from "./log.ts";
+import { AsyncLocalStorage } from "node:async_hooks";
+import process from "node:process";
+
+export const debugOption = object("debug", {
```

Comment:
More appropriate labeling (it's shown on the `--help` message):

```suggestion
export const debugOption = object("Global options", {
```